File: DWSIM.ProFeatures/FormGHG.cs

using DWSIM.Interfaces;
using System;
using System.Diagnostics;

namespace DWSIM.ProFeatures
{


    public partial class FormGHG : WeifenLuo.WinFormsUI.Docking.DockContent
    {

        public IFlowsheet CurrentFlowsheet;

        public FormGHG()
        {
            InitializeComponent();
            lblFeature = _lblFeature;
            _lblFeature.Name = "lblFeature";
        }

        private void FormGHG_Load(object sender, EventArgs e)
        {

            ExtensionMethods.FormExtensions.ChangeDefaultFont(this);

        }

        private void Button2_Click(object sender, EventArgs e)
        {

            Functions.CreateTransitionObject(CurrentFlowsheet, "", "GHG Emissions", "", "", default);

            Functions.DisplayTransitionForm(CurrentFlowsheet, "GHG Emissions");

        }

    }
}

File: DWSIM.ProFeatures/FormGHG.vb
Imports DWSIM.Interfaces

Public Class FormGHG

    Inherits WeifenLuo.WinFormsUI.Docking.DockContent

    Public CurrentFlowsheet As IFlowsheet

    Private Sub FormGHG_Load(sender As Object, e As EventArgs) Handles MyBase.Load

        ExtensionMethods.ChangeDefaultFont(Me)

    End Sub

    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click

        Functions.CreateTransitionObject(CurrentFlowsheet, "", "GHG Emissions", "", "", Nothing)

        Functions.ProcessTransition(CurrentFlowsheet)

    End Sub

End Class